syntax = "proto3";

package PBDriverWish;

message PBDriverWish {
    float left_servo = 1;
    float right_servo = 2;
    float top_servo = 3;
}